README.md

```diff
@@ -1,4 +1,18 @@
-# fhir-exercises
+# Geting started without installation
+These exercises are implemented using both Python and R. 
+
+## To run Python/Jupyter Notebook without installation
+ - Click [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stevetsa/fhir-exercises/HEAD)
+ - Select the notebook (Exercises*) from the left panel
+
+## To run R/RStudio without installation
+ - Create an account or sign in at [RStudio Cloud](https://rstudio.cloud/)
+ - Click "New Project" and select from the dropdown menu "New Project from Git Repository"
+ - Enter URL of this repository (https://github.com/stevetsa/fhir-exercises) and click OK
```

Comment:
Same as above
```suggestion
 - Enter URL of this repository (https://github.com/mitre/fhir-exercises) and click OK
```

Reply:
MyBinder pointing to the main repo rather than the fork - modify instructions